katello_certs_tools/sslToolConfig.py

```diff
@@ -43,12 +43,6 @@
 BASE_SERVER_RPM_NAME = 'katello-httpd-ssl-key-pair'
 BASE_SERVER_TAR_NAME = 'katello-httpd-ssl-archive'
 
-LEGACY_CA_KEY_NAME = 'ca.key'
```

Comment:
`ca.key` is actually used later in the code, but without using the constant :eyes: